pyproject.toml

```diff
@@ -1,63 +1,67 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
 [tool.poetry]
 name = "biteme"
 version = "0.1.0"
 description = "biteme"
 authors = ["Will Frey <jfrey89@gmail.com>"]
-license = "MIT"
 readme = "README.md"
 
-classifiers = [
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-]
+classifiers = ["Typing :: Typed"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-
-more-itertools = "^8.7.0"
-requests = "^2.25.1"
-typer = { version = "^0.3.2", extras = ["all"] }
+pydantic = { version = "^1.8.2", extras = ["dotenv"] }
+requests = "^2.26.0"
+typer = { version = "^0.4.0", extras = ["all"] }
+platformdirs = "^2.4.0"
 
 [tool.poetry.dev-dependencies]
-autoflake = "^1.4"
-bandit = "^1.7.0"
-black = "^20.8b1"
-coverage = { version = "^5.5", extras = ["toml"] }
-flake8 = "^3.8.4"
-ipykernel = "^5.5.0"
-isort = "^5.7.0"
-mypy = "^0.812"
-nox = "^2020.12.31"
-nox-poetry = "^0.8.1"
-pylint = "^2.7.1"
-pytest = "^6.2.2"
-pytest-cov = "^2.11.1"
-pytest-mock = "^3.5.1"
-pytest-randomly = "^3.5.0"
-rope = "^0.18.0"
-wemake-python-styleguide = "^0.15.1"
+black = "^21.9b0"
+ipykernel = "^6.4.1"
+isort = "^5.9.3"
+mypy = "^0.910"
+pytest = "^6.2.5"
+pytest-cov = "^2.12.1"
+pytest-mock = "^3.6.1"
+rope = "^0.20.1"
+pytest-randomly = "^3.10.1"
 
 [tool.poetry.scripts]
 biteme = "biteme:cli"
 
+
 [tool.black]
+target-version = ["py38", "py39"]
```

Comment:
good reminder you can actually configure those tools here 👍

Reply:
It's my preferred place!